### ISSUE TITLE
Every Build After #1503- Players cannot place blocks nor pick up items.

### ISSUE THREAD

**Issue by aquagene** [MinimallyCorrect/TickThreading#441]
There's no errors, but with every build after #1503, players are almost completely unable to place blocks anywhere, nor pick up items on the floor.

**Comment by Nohau** [comment]
Holy crap ultimate Grief Protection! 

Can a /op Pickup or place blocks

**Comment by aquagene** [comment]
Nope, can't do either of those things myself as OP either.

**Comment by LunNova** [comment]
Duplicate of #434.
